test/unit/math/prim/mat/meta/VectorViewMvt_test.cpp

```diff
@@ -0,0 +1,51 @@
+#include <stan/math/prim/mat.hpp>
+#include <gtest/gtest.h>
+#include <iostream>
+#include <stdexcept>
+#include <vector>
+
+// Just test the use case where a single matrix is passed in to VectorViewMvt.
+// The case where a std::vector of matrices is passed in is broken and will be refactored.
+
+TEST(MetaTraits, VectorViewMvt_matrix_double) {
+  using stan::VectorViewMvt;
+  using Eigen::Matrix;
+  using Eigen::Dynamic;
+
+  Matrix<double,Dynamic,1> a(10);
+  for (size_t n = 0; n < 10; ++n)
+    a[n] = n;
+  VectorViewMvt<Matrix<double,Dynamic,1> > av(a);
```

Comment:
`Matrix<double, Dynamic, 1>` (spaces after commas) is just `Eigen::VectorXd` (similarly, the other way around is `Eigen::RowVectorXd` and when they're both `Dynamic`, `Eigen::MatrixXd`.